lib/maps/quads.txt

```diff
@@ -0,0 +1,12 @@
+HW|IM|IM|IM|IM|IM|IM|IM|IM|iM|HW|IM
```

Comment:
The game code isn't smart enough to gracefully handle any errors with the map, so `iM` will need to be fixed to `IM` before I can merge it. See https://github.com/JSJitsu/ai-battle-engine/blob/5a5e2406400135aa63c34f6065d883807539516c/index.js#L80 for the specific issue.

I created #25 if you're interested in working a related ticket.